pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/docker/logrotate/logrotate.go

```diff
@@ -16,52 +16,59 @@ package logrotate
 
 import (
 	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
-
 	"k8s.io/utils/pointer"
 )
 
 // Config returns the content for logrotate units and files.
 // Whenever logrotate is ran, this config will:
-//   - rotate the log file if its size is > 100Mb OR if one day has elapsed
+//   - rotate the log file if its size is > 100Mb (only in docker case) OR if one day has elapsed
```

Comment:
This pkg serves only for docker, right? Then I guess we don't need to specify explicitly. WDYT?

Reply:
ok done